FEM/rf_msp_new.h

```diff
@@ -136,6 +153,23 @@ class CSolidProperties
 	// Set value for solid reactive system - TN
 	void setSolidReactiveSystem(FiniteElement::SolidReactiveSystem reactive_system);
 
+	// Kelvin/Voigt mapping routines for 6D vectors
```

Comment:
i guess these mapping routines are nothing to do with material models. why not move them somewhere `SolidMath`?